### ISSUE TITLE
Wizard, diferent items for interact.

### ISSUE THREAD

**Issue by Silent-Dead** [CitizensDev/Citizens#404]
Hi, is possible to configure the wizard's to interact with different items? like feather with 1 wizard, golden hoe with other, etc.

**Comment by Sdrubs** [comment]
Yes,go to citizens.yml in your plugins folder and search (CTRL+F) Interact item,then you put the items ID
